GMG/Scenes/Home/HomeView.swift

```diff
@@ -282,7 +370,7 @@ extension HomeView {
                         in: RoundedRectangle(cornerRadius: 18)
                     )
             }
-            .frame(width: 80)
+            .frame(width: songCount == 0 ? 156 : 77)
```

Comment:
`songCount`를 직접 받아서 처리하는 것보다 생성할 때 처리해주는 것은 어떨까요?

```suggestion
            .frame(width: isExpanded ? 156 : 77)
```